{
    "affected": [
        {
            "package": {
                "ecosystem": "Debian:8",
                "name": "python2.7",
                "purl": "pkg:deb/debian/python2.7?distro=jessie&repository_url=http%3A%2F%2Fdeb.freexian.com%2Fextended-lts"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "2.7.9-2-ds1-1+deb8u9"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "database_specific": {
        "type": "security update"
    },
    "id": "ELA-598-1",
    "modified": "2022-04-14T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/ELA-598-1"
        },
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/CVE-2019-16935"
        },
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/CVE-2021-3177"
        },
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/CVE-2021-4189"
        }
    ],
    "related": [
        "CVE-2019-16935",
        "CVE-2021-3177",
        "CVE-2021-4189"
    ]
}